backend/module-campaigns/src/Podkrepibg.Campaigns.Infrastructure/Configuration/DependencyInjection.cs

```diff
@@ -14,9 +14,10 @@ public static IServiceCollection AddInfrastructure(this IServiceCollection servi
             services.AddScoped<IApplicationReadOnlyDbContext>(provider => provider.GetService<CampaignsContext>());
 
             services.AddDbContext<CampaignsContext>(
-                options => options.UseNpgsql(configuration.GetConnectionString("CampaignDb")));
+                options => options.UseNpgsql($"Host={configuration.GetSection("DB_HOST").Value};Port={configuration.GetSection("DB_PORT").Value};Database={configuration.GetSection("DB_NAME").Value};Username={configuration.GetSection("DB_USER").Value};Password={configuration.GetSection("DB_PASS").Value}"));
```

Comment:
Can it be like this one or it is better to move the building the string into separate class ?